manim/mobject/types/image_mobject.py

```diff
@@ -146,7 +179,7 @@ class ImageMobjectFromCamera(AbstractImageMobject):
     def __init__(self, camera, **kwargs):
         self.camera = camera
         self.pixel_array = self.camera.pixel_array
-        AbstractImageMobject.__init__(self, **kwargs)
+        AbstractImageMobject.__init__(self, scale_to_resolution=False, **kwargs)
```

Comment:
interesting.
These are changes I did in a previous pr.
Why are they occurring here?